specs/LaunchingRegisteredProtocols.md

```diff
@@ -0,0 +1,316 @@
+# Background
+
+We are exposing an event that will be raised when an attempt to launch a protocol that is registered with the OS (external protocol) is made. The host will be given the option to cancel the launch. Cancelling the launch gives the host the opportunity to hide the default dialog, display a custom dialog, and then launch the external protocol themselves.
+
+# Description
+
+This event will be raised before the external protocol launch occurs. When an attempt to launch an external protocol is made, the default dialog is displayed in which the user can select `Open` or `Cancel` if the host does not cancel the event.  The `NavigationStarting` event will be raised before the `LaunchingExternalProtocol` event, followed by the `NavigationCompleted` event. The `SourceChanged`, `ContentLoading`, and `HistoryChanged` events will not be raised when a request is made to launch an external protocol.

```

Comment:
It would be good to define what an external protocol launch is. Like, 

> When navigating to a URI the URI scheme determines how to handle the URI. Some schemes like http, and https, are resolved by WebView2 and the navigation is handled in the WebView2 while other URI schemes may be registered externally to the WebView2 with the OS by other applications. These are external protocols and are handled by launching the registered application with the URI.